Automattic/jetpack
# VS Code: set PHP version level to 7.2

In setting up VS Code, I updated the PHP version to 7.2, which edits the settings.json for the repo. Should we set this as part of the default settings file sent over?

## Proposed changes:

* Set VSC to PHP 7.2

### Other information:


## Jetpack product discussion


n/a

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Open in VS Code and see if the bottom-right indicates PHP 5.6 when opening a PHP file.
